src/NPlug/Interop/LibVst.IPlugView.cs

```diff
@@ -84,9 +84,9 @@ private static partial ComResult canResize_ToManaged(IPlugView* self)
             return Get(self).CanResize();
         }
 
-        private static partial ComResult checkSizeConstraint_ToManaged(IPlugView* self, ViewRect* rect)
+        private static partial ComResult checkSizeConstraint_ToManaged(IPlugView* self, ref ViewRect* rect)
```

Comment:
You should remove the ref. It is not necessary. The signature of the native code is a pointer, so we just have to do `ref *(ViewRectangoe*)rect`